portal/src/graphql/portal/PasswordsScreen.tsx

```diff
@@ -1,16 +1,21 @@
-import React, { useContext } from "react";
+import React, { useCallback, useContext } from "react";
 import { useParams } from "react-router-dom";
 import { FormattedMessage, Context } from "@oursky/react-messageformat";
 import { Pivot, PivotItem, Text } from "@fluentui/react";
 import cn from "classnames";
 
 import PasswordPolicySettings from "./PasswordPolicySettings";
-import ForgotPasswordSettings from "./ForgotPasswordSettings";
-import { useAppConfigQuery } from "./query/appConfigQuery";
-import { useUpdateAppConfigMutation } from "./mutations/updateAppConfigMutation";
+import ForgotPasswordSettings, {
```

Comment:
I think we should define template name in this file and import in Setting component as Screen is parent?